main/options/options.cc

```diff
@@ -611,10 +611,14 @@ buildOptions(const vector<pipeline::semantic_extension::SemanticExtensionProvide
                                  "Extra parent directories which contain package files. These paths use an underscore "
                                  "package-munging convention, i.e. 'Project_Foo'",
                                  cxxopts::value<vector<string>>(), "string");
-    options.add_options(section)("extra-package-files-directory-prefix-slash",
+    options.add_options(section)("extra-package-files-directory-prefix-slash-deprecated",
                                  "Extra parent directories which contain package files. These paths use a slash "
                                  "package-munging convention, i.e. 'project/foo'",
                                  cxxopts::value<vector<string>>(), "string");
+    options.add_options(section)("extra-package-files-directory-prefix-slash",
+                                 "Extra parent directories which contain package files. These paths are constant names "
+                                 "joined by slashes, i.e. 'Project/Foo'",
```

Comment:
Is it worth changing the example to include a CamelCase name like `Project/FooBar` so that people understand it's not going to need `Project/Foo_Bar` or something like that?